packages/web-components/src/components.d.ts

```diff
@@ -12088,6 +12088,10 @@ export namespace Components {
           * If provided and greater than `0`, displays an outlined number badge at the bottom right of the icon. Numbers above `9999` are abbreviated to `'10K'` or `'100K'` for numbers in the thousands, `'1.5M'` for millions, `'1.5B'` for billions, and uses `'∞'` for one trillion or higher.
          */
         "notifications": number;
+        /**
+          * The size of a chosen Astro icon. Can be 'extra-small', 'small', 'normal', 'large', 'auto' or any custom value ('30px', '1rem', '3.321em')
+         */
+        "size": string;
```

Comment:
maybe rename to `icon-size` instead? at first glance, I would think `size` would refer to the host element not the icon within. 0/10